samples/client/src/main/java/global/covesa/sdk/client/push/FakeApplicationServer.kt

```diff
@@ -201,11 +206,12 @@ class FakeApplicationServer(private val context: Context) {
             Log.w(TAG, "Not an instance of a PrivateKeyEntry")
             return null
         }
-        return Signature.getInstance("SHA256withECDSA").run {
+        val signature = Signature.getInstance("SHA256withECDSA").run {
             initSign(entry.privateKey)
             update(data)
             sign()
         }
+        return signature.convertSignatureDerToRawRS()
```

Comment:
I forgot to fix it here too: https://codeberg.org/UnifiedPush/android-example/commit/3551384fd91c1d38232ed1e315911e55c8228989

You don't need BouncyCastle